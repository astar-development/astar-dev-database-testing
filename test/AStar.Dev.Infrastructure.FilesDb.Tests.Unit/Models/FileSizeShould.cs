using AStar.Dev.Infrastructure.FilesDb.Models;
using AStar.Dev.Utilities;

namespace AStar.Dev.Infrastructure.FilesDb.Tests.Unit.Models;

public class FileSizeShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => FileSize.Create(1, 2, 3)
                   .ToJson()
                   .ShouldMatchApproved();
}
