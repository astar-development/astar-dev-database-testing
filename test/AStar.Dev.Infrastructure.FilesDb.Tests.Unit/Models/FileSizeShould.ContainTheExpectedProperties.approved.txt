{
  "fileLength": 1,
  "height": 2,
  "width": 3
}